Makefile

```diff
@@ -417,7 +417,7 @@ nxOMSGenerateInventoryMof:
 
 nxOMSPlugin:
 	rm -rf output/staging; \
-	VERSION="2.18"; \
+	VERSION="2.19"; \
```

Comment:
@gauravga @vrdmr Guys if you are bumping this version up then can you include the latest oms_configuration.rb plugin, oms_diag_lib.rb, out_oms_diag.rb plugin files as part of module update?